runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsPlugin.java

```diff
@@ -271,6 +293,12 @@ private TileObject findTileObject(Tile tile, int id)
 		}
 
 		final GameObject[] tileGameObjects = tile.getGameObjects();
+		final DecorativeObject tileDecorativeObject = tile.getDecorativeObject();
+
+		if (tileDecorativeObject != null)
```

Comment:
You need to check if the id of the object is the same as what was clicked otherwise if a GameObject exists on the same tile as a DecorativeObject you will tag the decorative instead of the GameObject.

An example of this would be a crate and a window on a wall occupying the same tile.

Reply:
Ah right, I'll get this sorted in the morning.